tools/build-osbs/build-osbs.sh

```diff
@@ -77,10 +76,7 @@ function check_for_required_envs()
             rhpam-controller | \
             rhpam-kieserver | \
             rhpam-process-migration | \
-            rhpam-smartrouter | \
-            rhdm-controller | \
-            rhdm-decisioncentral | \
-            rhdm-kieserver)
+            rhpam-smartrouter)
```

Comment:
@spolti @jakubschwan isn't the rhpam dashbuilder image missing here?

Reply:
it is, this PR is just for rhdm removal, I'll add the Dashbuilder component later.